data/cameras.xml

```diff
@@ -193,7 +193,9 @@
 			<Color x="1" y="1">BLUE</Color>
 		</CFA>
 		<Crop x="64" y="12" width="0" height="0"/>
-		<Sensor black="127" white="4000"/>
+		<Sensor black="130" white="4000"/>
+		<Sensor black="781" white="32768" iso_list="100"/>
```

Comment:
That whitelevel doesn't look right.

Reply:
I didn't think so either, so I shot another set and it came out the same way.  I looked at the generated DNG and that's what it had.  I just tested these numbers to see what would really happen.  It appears the default (127, 4000) works much better.  I'll change that back.